pkg/linode-bs/driver_test.go

```diff
@@ -2,25 +2,22 @@ package linodebs
 
 import (
 	"encoding/json"
+	"fmt"
 	"math/rand"
 	"net/http"
 	"net/http/httptest"
 	"os"
 	"path/filepath"
 	"reflect"
+	"strconv"
+	"strings"
 	"testing"
 	"time"
 
 	"github.com/container-storage-interface/spec/lib/go/csi"
 	linodeclient "github.com/linode/linode-blockstorage-csi-driver/pkg/linode-client"
-	"github.com/linode/linode-blockstorage-csi-driver/pkg/metadata"
 	mountmanager "github.com/linode/linode-blockstorage-csi-driver/pkg/mount-manager"
 
-	"strconv"
```

Comment:
Not for this PR, but I'm wondering if we should consider using `gci` for linting import order like we did on CAPL (though it seems we don't actually even have a .golangci.yml on this repo? 🤔): https://github.com/linode/cluster-api-provider-linode/commit/158b033326eb5ffbea4b6cc82d3f5324f9aec247

Reply:
Yeah, we definitely could!

Would you be willing to submit a PR with that change, or file an issue (or ticket) so that I don't forget to do it? :smile:

Reply:
I'll put it in a ticket 👍